pentasharp/Models/Entities/User.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace pentasharp.Models.Entities
+{
+    public enum UserRole
```

Comment:
create a separate folder Enums, and put UserRole there, this will help us in the future if we introduce more enums it will be easier to maintain